build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     compileOnly("io.papermc.paper:paper-api:1.21.1-R0.1-SNAPSHOT")
 
     // Skript
-    compileOnly(group: 'com.github.SkriptLang', name: 'Skript', version: '2.8.7') {
+    compileOnly(group: 'com.github.SkriptLang', name: 'Skript', version: '2.9.3') {
```

Comment:
I think we will want to stick with 2.8.7 to cover the wildest breadth of support here since there are going to be no other code changes in the next release.